ddtrace/__init__.py

```diff
@@ -5,11 +5,6 @@
 
 LOADED_MODULES = frozenset(sys.modules.keys())
 
-# Configuration for the whole tracer from file. Do it before anything else happens.
-from ddtrace.internal.native import _apply_configuration_from_disk
-
-_apply_configuration_from_disk()
-
```

Comment:
AFAIU the logger is configured before the config object is loaded. We can set `DD_TRACE_DEBUG=true` using stable config today, how will this work? Maybe we should update the logger code? At least we should reconfigure the logger once we get the stable config.

Reply:
Ahhh great point! The logger reads the `DD_TRACE_DEBUG` environment variable directly. Ideally all configurations should be loaded using `ddtrace.settings._core.get_config`. 

I opened this PR address this: https://github.com/DataDog/dd-trace-py/pull/12392